src/components/ConnectionConfig.tsx

```diff
@@ -305,10 +303,7 @@ export const ConnectionConfig: FC<ConnectionConfigProps> = (props: ConnectionCon
   );
 };
 
-function getStyles() {
```

Comment:
I don't really mind this style imo. It makes it ever so slightly easier to add other styles later. Or if you ever want to reference a theme or something you can. 

That said i agree it doesn't need to go through the useStyles2 hook. 

That said how you have it is also fine by me!